Moderate-Background-Image/js/app.js

```diff
@@ -0,0 +1,109 @@
+import {
+  createVonageMediaProcessor
+} from '@vonage/ml-transformers/dist/ml-transformers.es.js';
```

Comment:
```suggestion
} from '../node_modules/@vonage/ml-transformers/dist/ml-transformers.es.js';
```
for some reason, the demo wouldn't work in StackBlitz without this. Sure it would be fine if ran locally, but just incase